services/user.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+
+
+User = get_user_model()
```

Comment:
According to the 'Code Style' section of the checklist, it is best to use `get_user_model()` instead of `User`. You've imported `get_user_model`, but then you've assigned it to `User` and used `User` throughout the code. It would be better to use `get_user_model()` directly when you need to access the User model.